schema/tracer_provider.json

```diff
@@ -211,9 +214,6 @@
                     "minimum": 0
                 }
             },
-            "required": [
-                "endpoint"
```

Comment:
Endpoint is not actually required for zipkin since there's a well defined default value.